StarVMC/StarAgmlLib/AgMath.h

```diff
@@ -39,7 +39,6 @@ const Double_t keV    = 0.001 * MeV;
 #define dup _dup_
 
 #define tanf _tanf_
-#define fpos _fpos_
```

Comment:
The point of this definition is to hide the "fpos" function, so that it doesn't interfere with the declaration of the FPOS volume in the FMS geometry.  It was always a bit of a hack.

If the compiler doesn't see a conflict building the geometry, then I am happy with this change.

Reply:
Hopefully, it won't cause a problem during runtime when the geometry is loaded. Although, what you propose in #308 is still worrisome and may not play well with modern compilers
https://github.com/star-bnl/star-sw/pull/308/files/2fb56841b5cafae4a95e587a5f65acaf01428d3e#r882221155

Reply:
The FMS geometry contains a data structure named FPOS.  Under earlier versions of AgML, this resulted in an object named "fpos" being declared... causing a compiler error.  Later releases of AgML move this object into the namespace of the geometry module.  So... if it compiles without the define, we will be good.